test/test_transforms.py

```diff
@@ -1834,6 +1764,83 @@ def test_pad_with_mode_F_images(self):
         assert_equal(padded_img.size, [edge_size + 2 * pad for edge_size in img.size], check_stride=False)
 
 
+def _test_randomness(fn, trans, configs):
+    random_state = random.getstate()
+    random.seed(42)
+    img = transforms.ToPILImage()(torch.rand(3, 16, 18))
+
+    for p in [0.5, 0.7]:
+        for config in configs:
```

Comment:
Looks like we don't even test on more than one config for a given function, so maybe we can remove this for loop, and change `[{}]` into `{}` (and related changes)